docs/cli/kops_completion.md

```diff
@@ -0,0 +1,38 @@
+## kops completion
+
+Outputs kops shell completion for the given shell (bash)
+
+### Synopsis
+
+
+Outputs kops shell completion for the given shell (bash)
+
+```
+kops completion
+```
+
+### Options
+
+```
+      --shell string   target shell (bash).
```

Comment:
Done and done. Have added an example to the completion command and created a new issue here - https://github.com/kubernetes/kops/issues/841